ext/standard/net.c

```diff
@@ -291,6 +293,10 @@ PHP_FUNCTION(net_get_interfaces) {
 		                     p->ifa_addr, p->ifa_netmask,
 		                     (p->ifa_flags & IFF_BROADCAST) ? p->ifa_broadaddr : NULL,
 					         (p->ifa_flags & IFF_POINTOPOINT) ? p->ifa_dstaddr : NULL);
+		status = zend_hash_str_find(Z_ARR_P(iface), "up", sizeof("up") - 1);
+		if (!status) {
+			add_assoc_bool_ex(iface, "up", sizeof("up") - 1, ((p->ifa_flags & IFF_UP) != 0));
+		}
```

Comment:
Just doing the add_assoc_bool_ex without the lookup beforehand should be fine here. Can also use add_assoc_bool and save the `sizeof("up") - 1` argument.

Reply:
Was not sure about the "importance" of this new entry so I put in the end :-)